liquibase-core/src/test/groovy/liquibase/changelog/DatabaseChangeLogTest.groovy

```diff
@@ -494,4 +500,30 @@ create view sql_view as select * from sql_table;'''
         "D:\\a\\liquibase\\DBDocTaskTest.xml" | "a/liquibase/DBDocTaskTest.xml"
     }
 
+    def "warning message is logged when changelog include fails because file does not exist"() {
```

Comment:
I would like us to add a test for `ChangeLogParserConfiguration.MissingIncludeConfiguration.FAIL`.  I realize this should result in the default/existing behavior but without a test we cannot be sure that remains true over time.

CC @mhewedy @MalloD12

Reply:
Updated existent `"include fails if file does not exist"` test to assert `ChangeLogParserConfiguration.ON_MISSING_INCLUDE_FILE.getCurrentValue() == ChangeLogParserConfiguration.MissingIncludeConfiguration.FAIL`

Thanks Erz!